Src/Base/AMReX_GpuDevice.cpp

```diff
@@ -358,6 +371,11 @@ Device::Finalize ()
     cudaProfilerStop();
 #endif
 
+#if defined(AMREX_USE_HIP) && defined(AMREX_USE_ROCTX)
+    roctracer_stop();
```

Comment:
I suggest we move roctracer_stop to `Device::profilerStop`, remove the call to `cudaProfilerStop` above, and call `Device::profileStop()`.